solutions/3527/01.py

```
@@ -0,0 +1,20 @@
+from collections import defaultdict
+
+
+class Solution:
+    def findCommonResponse(self, responses: List[List[str]]) -> str:
```

Ruff (0.14.10)

Undefined name `List`

(F821)